### PR TITLE
3MFLoader: Add Preliminary Support for Loading Implicit Functions

### PR DESCRIPTION
**Pull request by zalo**
Related issue: [#29548](https://github.com/mrdoob/three.js/issues/29548)

tl;dr: This PR adds basic `implicitfunction` (parsing only!) functionality to three.js's 3MFLoader.

**Context**

3MF files can now contain volumetric implicits defined as operation execution graphs.

The 3MF consortium has released a 3D Modeller for generating .3mf’s with them, similar to nTopology:
https://github.com/3MFConsortium/gladius

This presents an opportunity to make SDF Implicits a first-class feature of 3D on the web!

**Description:**

This PR advances this goal by adding a parser that exposes this new information to three.js.

Eventually, when TSL and WebGPU are mature, this information can be used to construct a TSL nodegraph, which can be used to import and visualize these 3MF implicit functions.

You can test this PR easily by going [here](https://raw.githack.com/zalo/three.js/feat-3mf-volumetric/examples/webgl_loader_3mf.html), selecting `volumetric` from the drop-down, and observing the implicit tree that was output to the console.

### CODE REVIEW

**Review by Mugen87 — APPROVED** [review]

**Comment by linbingquan** [comment]
图片

It reported an error on my computer...

**Comment by Mugen87** [comment]
In this case I would simply revert the changes to the example to keep things simple.

**Comment by Mugen87** [comment]
https://github.com/mrdoob/three.js/commit/8104618a31e9fa37cbe7a483fa3b38318bf63220
